cloudfront/lambdas/tsconfig.build.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "noEmit": false
+  },
+  "exclude": ["node_modules","**/*.test.ts", "**/test_*.ts"]
```

Comment:
fwiw, credit where it's due, if you've copied this idea of a `tsconfig.build.json` extending a base config from the example in weco.org - I copied it off digirati!